proto/device.proto

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
please include in one of the other proto files -- the granularity of "device" is too small.  Rather, things should be grouped by logical bucket for different proto files, and then the "message" scope can be granular.

Reply:
Moved into network_state.proto